tests/src/whisk/core/controller/test/RespondWithHeadersTests.scala

```diff
@@ -44,47 +62,223 @@ class RespondWithHeadersTests extends ControllerTestCommon with RespondWithHeade
 
     behavior of "General API"
 
-    val routes = {
-        pathPrefix("api" / "v1") {
-            sendCorsHeaders {
-                path("one") {
-                    complete(OK)
-                } ~ path("two") {
-                    complete(OK)
-                } ~ options {
-                    complete(OK)
-                } ~ reject
-            }
-        } ~ pathPrefix("other") {
-            complete(OK)
```

Comment:
These tests should've stayed as is. The new ones are no "Unit"-Tests but rather integration tests for the controller, as they need a deployed version of it. The new tests should go into a different package, `services` might be a good fit.